LobbyServer2/BridgeServer/BridgeServerProtocol.cs

```diff
@@ -591,7 +603,7 @@ public void CheckIfAllSelected()
                         && playerInfo.ReadyState != ReadyState.Ready)
                     {
                         CharacterType randomType = account.AccountComponent.LastCharacter;
-                        if (account.AccountComponent.LastCharacter == playerInfo.CharacterType)  // TODO it does not automatically mean that currently selected character is allowed
+                        if (account.AccountComponent.LastCharacter == playerInfo.CharacterType)
```

Comment:
It's not about that. If a player switched character it does not automatically mean it's a legal switch e.g. two players picked the same character simultaneously